au/code/au/magnitude_test.cc

```diff
@@ -113,6 +147,11 @@ TEST(IntegerPart, PicksOutIntegersFromNumerator) {
     EXPECT_EQ(integer_part(PI * sqrt(mag<32>()) / mag<15>()), mag<4>());
 }
 
+TEST(IntegerPart, PreservesSign) {
+    EXPECT_EQ(integer_part(-mag<1>()), -mag<1>());
+    EXPECT_EQ(integer_part(-mag<8765>()), -mag<8765>());
```

Comment:
I personally don't have a problem mixing the `EXPECT_THAT` matcher syntax with the legacy `EXPECT_EQ` assertion syntax (in that I'll write new tests in the new syntax leaving existing alone, but I do understand the desire to maintain consistency.

Reply:
This is a bit of a time capsule.  I wrote this back in December, which is probably before you persuaded me to use matchers consistently!  I'll update these new test cases.  At some point, we should migrate the whole repo, too.

Reply:
I'm also fine not touching time capsule tests.  Sometimes it's nice to be looking at history and say "nice, this code has been here doing it's job correctly for X number of years." 😁